library/library-template.html

```diff
@@ -22,11 +22,11 @@
     <div class="top-container">
         <!--vue-ssr-outlet-->
         <div id="content-about" style="font-size: 0.85rem; line-height: 1.2; padding: 10px; text-align: center; background-color: beige;">
-            මෙහි ඇතුලත් කර ඇති පොත් බොහොමයක් පැරණි බෞද්ධ පොත් වන අතර ධර්ම දානය සඳහා ලියන ලද පොත් වේ. කතෘ ජිවත්ව සිටින
+            මෙහි ඇතුළත් කර ඇති පොත් බොහොමයක් පැරණි බෞද්ධ පොත් වන අතර ධර්ම දානය සඳහා ලියන ලද පොත් වේ. කතෘ ජිවත්ව සිටින
             අවස්ථාවලදී කතු අවසරය සහිතව අඩවියට එක් කිරීමට උත්සාහ දරා ඇත. ඔබ කතෘ වන යම් පොතක් මෙම අඩවියෙන් ඉවත් කර ගැනීමට අවශ්‍ය නම් හෝ 
-            මෙහි නොමැති පොත් අඩවියට ධර්ම දානය පිණිස ඇතුළු කිරීමට අප වෙත ලියන්න admin <i class="fas fa-at"></i> pitaka.lk.<br>
-            යතුරුලියනය කල පොත් ඇසුරින් සැකසු අන්තර්ජාලය අනවශ්‍ය offline මෘදුකාංග <i class="far fa-laptop"></i> <a href="https://tipitaka.lk/library/download/448">පරිඝනක සඳහා</a> සහ
-            <i class="fab fa-android"></i> <a href="https://play.google.com/store/apps/details?id=lk.pitaka.books">Android දුරකථන සඳහා</a> ලබාගන්න.<br>
+            මෙහි අඩංගු නොවන පොත් අඩවියට ධර්ම දානය පිණිස ඇතුළු කිරීමට අප වෙත ලියන්න admin <i class="fas fa-at"></i> pitaka.lk.<br>
+            යතුරුලියනය කළ පොත් ඇසුරින් සැකසු අන්තර්ජාලය අනවශ්‍ය (offline) මෘදුකාංග <i class="far fa-laptop"></i> <a href="https://tipitaka.lk/library/download/448">පරිගනක සඳහා</a> සහ
+            <i class="fab fa-android"></i> <a href="https://play.google.com/store/apps/details?id=lk.pitaka.books">ඇන්ඩ්‍රොයිඩ් දුරකථන සඳහා</a> බාගන්න.<br>

```

Comment:
පරිගණක is the correct word according to Sinhala dictionaries right? 
බාගත කරගන්න/භාගත කරගන්න sounds more formal on the website.